### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by brandon9838** [wangyida/softpool#6]
Hi, I'm trying to reproduce the result by training softpoolnet, 
but it seems that train.py, model.py are a mixture of softpoolnet, grnet, and pointnet. The latter 2 works are not included in the architecture reported in your paper.

If I only want to reproduce the completion result of softpoolnet, can I comment out output3, output4, part_regions, emd3, emd4 in train.py(line 115), and the associated networks in model.py?

By the way, are there tensorflow codes for softpoolnet in the tensorflow folder?

Thanks for replying!

**Comment by wangyida** [comment]
Hi yep you can comment out those output produced for grnet and foldingnet, together with architectures in model.py. We have softpool operaters in tensorflow folder, the softpoolnet architecture is not identical to our paper in TensorFlow's implementation.

Notice that the emd loss function there should be either earth moving distance or chamfer distance
